src/System Application/App/Extension Management/src/ExtensionManagement.Page.al

```diff
@@ -366,6 +383,7 @@ page 2500 "Extension Management"
         // Determining Record and Styling Configurations
         IsInstalled := ExtensionInstallationImpl.IsInstalledByPackageId(Rec."Package ID");
         IsTenantExtension := Rec."Published As" <> Rec."Published As"::Global;
+        IsSourceSpecificationAvailable := StrLen(Rec."Source Repository Url") > 0;
```

Comment:
When is this set? Seems like anyone would be able to click this as long as the `url` is set. Is that how this should be? Not hidden behind admin permissions etc?